numpy/core/src/multiarray/mapping.c

```diff
@@ -727,8 +727,14 @@ prepare_index(PyArrayObject *self, PyObject *index,
         }
     }
     else if (used_ndim > PyArray_NDIM(self)) {
+	char error_message[50];
```

Comment:
nitpick: make sure the indentation is consistent. IIRC, we use 4 space for indentation in the C code.